rmf_demo_assets/models/Hedge/model.sdf

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='utf-8'?>
+<sdf version="1.6">
+  <model name="Hedge">
```

Comment:
![image](https://user-images.githubusercontent.com/5383623/76713492-e03cfe80-675b-11ea-8161-5ada543575c3.png)
The hedges seem to be bit unnaturally green